lib/code_corps/analytics/segment_traits_builder.ex

```diff
@@ -78,10 +78,12 @@ defmodule CodeCorps.Analytics.SegmentTraitsBuilder do
     }
   end
   defp traits(%CodeCorps.ProjectUser{} = record) do
-    record = record |> Repo.preload(:project)
+    record = record |> Repo.preload([:project, :user])
     %{
       project: record.project.title,
-      project_id: record.project_id
+      project_id: record.project_id,
+      member_id: record.user.id,
+      member: record.user.username
```

Comment:
Would reverse these to keep with consistency of below: `member` and then `member_id`.